app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -644,6 +644,10 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
         align-items: center;
       }
 
+      @media screen and (max-width: $breakpoint-xs) {
```

Comment:
I think this is definitely looking much better, but it's bugging me that the alignment between the checkboxes and the text is a little off and we have quite a big gap between the lines when it wraps 🤔 

I had an experiment and if we remove this media query, and the one above on line 643, and instead add:

```
flex-wrap: wrap;
align-items: center;
```

to the main CSS, I think we have something that works for all screen sizes:

https://user-images.githubusercontent.com/20773163/181005111-2c37fdfc-ad76-4dc6-be98-4e7bba7e0d9d.mp4

What do you think?

Reply:
Fantastic! :+1: